pkgs/applications/networking/cluster/waypoint/default.nix

```diff
@@ -77,7 +76,7 @@ buildGoModule rec {
       through a consistent and repeatable workflow.
     '';
     license = licenses.mpl20;
-    maintainers = with maintainers; [ winpat jk ];
-    platforms = platforms.linux;
+    maintainers = with maintainers; [ winpat jk techknowlogick ];
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
```suggestion
```

Maybe we don't need to set this if default platforms work (#153464).

Reply:
fixed, thanks for pointer :)